### ISSUE TITLE
Harbor 1.10 - Notary server/signer will not start

### ISSUE THREAD

**Issue by ThomasRasmussen** [goharbor/harbor#12305]
If you are reporting a problem, please make sure the following information are provided:

**Expected behavior and actual behavior:**
Our Harbor installation has been migrated from very early versions (I believe from v 1.5 to now v 1.10.1)
We previously had notary server disabled as we could not get it to work, now we want to enable it, so we have shut down harbor, run 
`install.sh --with-notary --with-clair --with-chartmuseum`

Notary server starts but will restart every minut or so.
The logfile notary-server.log continues to log this:
```
Jun 22 09:31:16 172.23.0.1 notary-server[1756]: error: file does not exist
Jun 22 09:31:16 172.23.0.1 notary-server[1756]: waiting for database to come up.
Jun 22 09:31:17 172.23.0.1 notary-server[1756]: error: file does not exist
Jun 22 09:31:17 172.23.0.1 notary-server[1756]: notaryserver database failed to come up within 30 seconds
Jun 22 09:31:19 172.23.0.1 notary-server[1756]: 2020/06/22 07:31:19 Updating database.
Jun 22 09:31:19 172.23.0.1 notary-server[1756]: 2020/06/22 07:31:19 schema_migrations table does not require update, skip.
Jun 22 09:31:19 172.23.0.1 notary-server[1756]: error: file does not exist
Jun 22 09:31:19 172.23.0.1 notary-server[1756]: waiting for database to come up.
```

I have tried to run the script https://github.com/goharbor/harbor/blob/master/tools/notary-migration-fix.sh but this changes nothing (DELETE 0).

**Steps to reproduce the problem:**
Upon all restarts of harbor, the error persists.

**Versions:**

- harbor version: 1.10.1
- docker engine version: 18.09.6
- docker-compose version: 1.25.0

**Comment by ThomasRasmussen** [comment]
Should maybe mention that an earlier version of harbor we tried to test notary but as we could not get it to work, we disabled it again, and then have migrated to later versions. Maybe this causes the error.

There is no images signed by the notary, so if there are directions on how to completely clean out the notary files/configuration/database, this should be possible.

I have tried to delete the notaryserver and notarysigner database from postgres, but this end in fatal errors, the databases are not recreated after starting harbor.
